### PR TITLE
project vet class create

### PR DESCRIPTION
**Pull request by rukshand2001**

### CODE REVIEW

File: src/main/java/com/rukshan/petcare/vet/VetController.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class VetController {
+        @Autowired
+        private VetServices vetservices;
+
+        @GetMapping("/vet_ping")
```
Comment (aselarbd) [inline src/main/java/com/rukshan/petcare/vet/VetController.java]:
```suggestion
        @GetMapping("/vet/ping")
```

File: src/main/java/com/rukshan/petcare/vet/VetController.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        public Message vet_ping(){
+            return vetservices.vet_ping();
```
Comment (aselarbd) [inline src/main/java/com/rukshan/petcare/vet/VetController.java]:
```suggestion
            return vetservices.ping();
```

File: src/main/java/com/rukshan/petcare/vet/VetController.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package com.rukshan.petcare.vet;
+import com.rukshan.petcare.utils.Message;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class VetController {
+        @Autowired
+        private VetServices vetservices;
```
Comment (aselarbd) [inline src/main/java/com/rukshan/petcare/vet/VetController.java]:
```suggestion
        private VetServices vetServices;
```

File: src/main/java/com/rukshan/petcare/vet/VetServices.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package com.rukshan.petcare.vet;
+
+import com.rukshan.petcare.utils.Message;
+import org.springframework.stereotype.Service;
+
+@Service
+public class VetServices {
+    public Message vet_ping(){
```
Comment (aselarbd) [inline src/main/java/com/rukshan/petcare/vet/VetServices.java]:
```suggestion
    public Message ping(){
```

File: src/main/java/com/rukshan/petcare/vet/VetServices.java [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+        /* calling ping function */
+
+        return new Message("Hello vet",300);
```
Comment (aselarbd) [inline src/main/java/com/rukshan/petcare/vet/VetServices.java]:
```suggestion
        return new Message("Hello vet",200);
```

File: src/main/java/com/rukshan/petcare/vet/VetController.java (hunk 1 above)
Comment (rukshand2001) [reply src/main/java/com/rukshan/petcare/vet/VetController.java]: corrected

File: src/main/java/com/rukshan/petcare/vet/VetController.java (hunk 2 above)
Comment (rukshand2001) [reply src/main/java/com/rukshan/petcare/vet/VetController.java]: corrected

File: src/main/java/com/rukshan/petcare/vet/VetController.java (hunk 3 above)
Comment (rukshand2001) [reply src/main/java/com/rukshan/petcare/vet/VetController.java]: corrected

File: src/main/java/com/rukshan/petcare/vet/VetServices.java (hunk 4 above)
Comment (rukshand2001) [reply src/main/java/com/rukshan/petcare/vet/VetServices.java]: changed

File: src/main/java/com/rukshan/petcare/vet/VetServices.java (hunk 5 above)
Comment (rukshand2001) [reply src/main/java/com/rukshan/petcare/vet/VetServices.java]: changed

File: src/main/java/com/rukshan/petcare/vet/VetController.java [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class VetController {
+        @Autowired
+        private VetServices vetServices;
+
+        @GetMapping("vet/ping")
```
Comment (aselarbd) [inline src/main/java/com/rukshan/petcare/vet/VetController.java]:
```suggestion
        @GetMapping("/vet/ping")
```

**Review by aselarbd — APPROVED** [review]

**Pull request merged**